.github/workflows/ci.yml

```diff
@@ -9,19 +9,11 @@ jobs:
     name: "RSpec / Ruby ${{ matrix.ruby }}"
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        ruby:
-          [
-            "2.3",
-            "2.4",
-            "2.5",
-            "2.6",
-            "2.7",
-            "3.0",
-            "3.1",
-            "3.2"
-          ]
+        ruby: ["2.3", "2.4", "2.5", "2.6", "2.7", "3.0", "3.1", "3.2", "3.3"]
     steps:
+      - run: sudo apt-get install libcurl4-openssl-dev
```

Comment:
Do we need to explicitly install `libcurl4-openssl-dev` for new version of `em-http-request` to work?

Reply:
Also, it seems to be a dev dependency wdyt

Reply:
Looked at https://github.com/igrigorik/em-http-request/pull/359#issue-2065876700. But, you can cross check 👍

Reply:
Yeah, I didn't look in detail at the contents of https://github.com/igrigorik/em-http-request/pull/359; its contents looked harmless enough and I was happy to blindly accept them.

Reply:
Got it